prow/config.yaml

```diff
@@ -14893,7 +14893,7 @@ periodics:
       - name: USER
         value: prow
       - name: JENKINS_AWS_CREDENTIALS_FILE
-        value: /etc/aws-cred/aws-cred
+        value: /etc/aws-cred/credentials
```

Comment:
Did you mean to modify the `-canary` job here?

Reply:
I changed the data name from `aws-cred` to `credentials`, so that's also need to be updated I think.